proto/babylon/btcstaking/v1/btcstaking.proto

```diff
@@ -195,4 +196,11 @@ message SelectiveSlashingEvidence {
     // the covenant adaptor/Schnorr signature pair. It is the consequence
     // of selective slashing.
     bytes recovered_fp_btc_sk = 3;
-  }
+}
+
+message InclusionProof {
```

Comment:
Message needs doc comments as otherwise `make proto-lint` won't pass.

A bit tangent, I am not sure how `make proto-lint` got out of CI, most probably during migration to re-usable pipelines.